tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -317,7 +317,18 @@ def check_succeed(self, validate_metrics=True):
             self.raise_on_bad_log_lines(node)
         # Generate charts from the result
         self.logger.info(f"Generating charts with command {self.chart_cmd}")
-        metrics = json.loads(self.node.account.ssh_output(self.chart_cmd))
+        self.node.account.ssh_output(self.chart_cmd)
```

Comment:
So we still get the chart in the artifact, I guess?

Reply:
Yep correct